lib/httparty/connection_adapter.rb

```diff
@@ -195,5 +197,11 @@ def attach_ssl_certificates(http, options)
         end
       end
     end
+
+    def normalize_key_files options
```

Comment:
I don't understand what this is doing. Is it changing certificate options by reading files somewhere on the filesystem? 

Could you perhaps add a test or two to show how this works and ensure that I don't break it in the future. These options are not something I use regularly, so I want to make sure I don't mess it up going forward.